sdk/include/psdk/shellapi.h

```diff
@@ -646,6 +646,13 @@ SHTestTokenMembership(
     _In_ ULONG ulRID);
 #endif
 
+HRESULT WINAPI
+SHEnumerateUnreadMailAccountsW(
```

Comment:
Missing function SHEnumerateUnreadMailAccountsA in the PR.

Reply:
There is no `SHEnumerateUnreadMailAccountsA` for Win2k3/WinXP.
MS Learn is wrong.

Reply:
Confirmed on WinXPSP3 at least:
![VirtualBox_Windows XP_19_01_2025_14_17_27](https://github.com/user-attachments/assets/904ed368-49f0-42f4-9f1d-44877df8d5c5)

Reply:
`...W` exists.
`...A` doesn't exist.